File: baekjoon/1100_1199/1110/1110.py
import sys
input = sys.stdin.readline

input_n = int(input())
n = input_n
count = 0

while True:
    nl = n // 10
    nr = n % 10
    n = (nr*10) + (nl+nr)%10

    count += 1
    if n == input_n: break

print(count)

File: baekjoon/1100_1199/1110/1110.c
#include <stdio.h>

int main(){
  int n, input_n, count = 0;
  scanf("%d", &input_n);
  n = input_n;

  do{
    int nl, nr;

    nl = n / 10;
    nr = n % 10;
    n = nr*10 + (nl+nr)%10;

    count++;
  } while(n != input_n);

  printf("%d", count);
  return 0;
}
